### PR TITLE
Fix #117: make rel-bot work with setuptools-scm

### LINKED ISSUE #117: make release-bot compatible with setuptools-scm plugin

### PR DESCRIPTION
**Pull request by TomasTomecek**
TODO:
* [x] implement updating release changelog even when released on github
  * [x] get the changelog first and compare
* [x] after the github release is done, check out the tag
* [x] update documentation

### CODE REVIEW

**Comment by TomasTomecek** [comment]
This is now done. I made this release with the code:
https://github.com/TomasTomecek/ansible-bender/releases/tag/0.3.2
https://github.com/TomasTomecek/ansible-bender/pull/44 (comment about releasing to gh is missing b/c I was still working on the code)

**Comment by TomasTomecek** [comment]
Nice, tests passed, ready for review.

File: release_bot/github.py [hunk 1]
```diff
@@ -305,51 +310,37 @@ def make_new_release(self, new_release):
         if response.status_code != 201:
             msg = f"Failed to create new release on github:\n{response.text}"
             raise ReleaseException(msg)
+        return True, new_release
+
+    def update_changelog(self, new_version):
+        self.git.fetch_tags()
+        self.git.checkout(new_version)
+        # FIXME: make the file name configurable
+        p = os.path.join(self.git.repo.name, "CHANGELOG.md")
```
Comment (jpopelka) [inline release_bot/github.py]:
```suggestion
        p = os.path.join(self.git.repo_path, "CHANGELOG.md")
```

File: release_bot/releasebot.py [hunk 2]
```diff
@@ -35,8 +36,11 @@ class ReleaseBot:
 
     def __init__(self, configuration):
         self.conf = configuration
-        self.github = Github(configuration)
-        self.pypi = PyPi(configuration)
+        url = f'https://github.com/{self.conf.repository_owner}/{self.conf.repository_name}.git'
+        self.git = Git(url, self.conf)
+
```
Comment (jpopelka) [inline release_bot/releasebot.py]:
```suggestion
```

**Comment by TomasTomecek** [comment]
Thank you both for review, @jpopelka I accepted your suggestions. Feel free to merge.

**Pull request merged**